recipes/dfp/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import copy, get
+
+required_conan_version = ">=1.54.0"
+
+
+class DfpConan(ConanFile):
+    name = "dfp"
+    package_type = "library"
+
+    # Optional metadata
+    description = "Decimal Floating Point Arithmetic Library"
+    homepage = "https://github.com/epam/DFP/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ("Apache-2.0", "Intel")
+    topics = ("decimal", "dfp", "ieee-754", "deltix")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
```

Comment:
Conan recipes usually allow to modify position independent code via the `fPIC` option. I see in your upstream CMakeLists that [it's always turned ON](https://github.com/epam/DFP/blob/master/native/CMakeLists.txt#L76), but unless not supported by the library, we usually prefer to give the end user the option to enable-disable it by themsevlves.

Let me know if it's ok to add the option and I'll do that myself :)